fvm/serviceaccount.go

```diff
@@ -7,15 +7,13 @@ import (
 )
 
 const systemChunkTransactionTemplate = `
-import FlowServiceAccount from 0x%s
+import FlowEpoch from 0x%s
+
 transaction {
   prepare(serviceAccount: AuthAccount) { 
-
-  }
-
-  execute {
-    // TODO: replace with call to service account heartbeat
- 	log("pulse")
+	let heartbeat = serviceAccount.borrow<&FlowEpoch.Heartbeat>(from: FlowEpoch.heartbeatStoragePath)
+      ?? panic("Could not borrow heartbeat from storage path")
+    heartbeat.advanceBlock()
```

Comment:
Is there any way we can tests if this works correctly?

Reply:
We have tests for the smart contract business logic using the emulator. Do you mean a test that the system chunk calls the `advanceBlock` function at all?